spec/fortnox/api/types/housework_types_spec.rb

```diff
@@ -42,49 +45,46 @@
         it 'raises an error' do
-          is_expected.to raise_error(Fortnox::API::RemoteServerError, error_message)
+          expect(subject).to raise_error(Fortnox::API::RemoteServerError, error_message)
         end
       else
         it { is_expected.not_to raise_error }
       end
     end
   end
 
-  TYPE_ROT = Fortnox::API::Types::TaxReductionTypes['rot']
-  TYPE_RUT = Fortnox::API::Types::TaxReductionTypes['rut']
-
-  it_behaves_like 'housework type', 'CONSTRUCTION', TYPE_ROT
-  it_behaves_like 'housework type', 'ELECTRICITY', TYPE_ROT
-  it_behaves_like 'housework type', 'GLASSMETALWORK', TYPE_ROT
-  it_behaves_like 'housework type', 'GROUNDDRAINAGEWORK', TYPE_ROT
-  it_behaves_like 'housework type', 'MASONRY', TYPE_ROT
-  it_behaves_like 'housework type', 'PAINTINGWALLPAPERING', TYPE_ROT
-  it_behaves_like 'housework type', 'HVAC', TYPE_ROT
-  it_behaves_like 'housework type', 'OTHERCOSTS', TYPE_ROT, housework: false
-
-  it_behaves_like 'housework type', 'MAJORAPPLIANCEREPAIR', TYPE_RUT
-  it_behaves_like 'housework type', 'MOVINGSERVICES', TYPE_RUT
-  it_behaves_like 'housework type', 'ITSERVICES', TYPE_RUT
-  it_behaves_like 'housework type', 'CLEANING', TYPE_RUT
-  it_behaves_like 'housework type', 'TEXTILECLOTHING', TYPE_RUT
-  it_behaves_like 'housework type', 'SNOWPLOWING', TYPE_RUT
-  it_behaves_like 'housework type', 'GARDENING', TYPE_RUT
-  it_behaves_like 'housework type', 'BABYSITTING', TYPE_RUT
-  it_behaves_like 'housework type', 'OTHERCARE', TYPE_RUT
-  it_behaves_like 'housework type', 'OTHERCOSTS', TYPE_RUT, housework: false
+  it_behaves_like 'housework type', 'CONSTRUCTION', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'ELECTRICITY', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'GLASSMETALWORK', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'GROUNDDRAINAGEWORK', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'MASONRY', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'PAINTINGWALLPAPERING', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'HVAC', 'TYPE_ROT'
+  it_behaves_like 'housework type', 'OTHERCOSTS', 'TYPE_ROT', housework: false
+
+  it_behaves_like 'housework type', 'MAJORAPPLIANCEREPAIR', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'MOVINGSERVICES', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'ITSERVICES', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'CLEANING', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'TEXTILECLOTHING', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'SNOWPLOWING', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'GARDENING', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'BABYSITTING', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'OTHERCARE', 'TYPE_RUT'
+  it_behaves_like 'housework type', 'OTHERCOSTS', 'TYPE_RUT', housework: false
 
   # rubocop:disable RSpec/RepeatedExample
   # rubocop:disable RSpec/RepeatedDescription
   pending 'to be added' do
-    raise Exception, 'Will be supported 2021-01-01'
+    raise StandardError, 'Will be supported 2021-01-01'
```

Comment:
Rubocop suggested changing from Exception to StandardError

Reply:
Do we need this anymore? The "Will be supported" date has passed ...

Reply:
Ye, I need to implement it first 😅 Will do in a follow up PR...